docs/sources/flow/concepts/config-language/_index.md

```diff
@@ -1,14 +1,20 @@
 ---
 aliases:
+- /docs/grafana-cloud/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/send-data/agent/flow/concepts/config-language/
+- concepts/configuration-language/
```

Comment:
Is this intended to redirect the old page (/docs/agent/<AGENT_VERSION>/flow/config-language/) to the new page (/docs/agent/<AGENT_VERSION>/flow/concepts/config-language/)? 

I tested locally and there isn't a redirect for this. I would expect something like:

```suggestion
- ../config-language/ # /docs/agent/<AGENT_VERSION>/flow/config-language/
```

Reply:
I've added this with comments and each alias should be in the right place now.